docs/en/02_Developer_Guides/00_Model/10_Versioning.md

```diff
@@ -308,7 +309,19 @@ class Banner extends Page
 }
 ```
 
-Note that ownership cannot be used with polymorphic relations. E.g. has_one to non-type specific `DataObject`. 
+Note that ownership cannot be used with polymorphic relations. E.g. has_one to non-type specific `DataObject`.
+
+#### Unversioned dataobject ownership
+
+Ownership can be used with non-versioned dataobjects, as the necessary functionality is included by default
+by the versioned object through the `[api:SilverStripe\Versioned\RecursivePublishable]` extension which is
+applied to all objects.
+
+However, it is important to note that even when saving un-versioned objects, it is necessary to use
+`->publishRecursive()` to trigger a recursive publish.
+
+As long as the `owns` config is specified on all elements in the tree, you can use any combination of
+versioned or unversioned dataobjects.
```

Comment:
Does this mean that we don't need to worry about `->publishRecursive()` if we have an `owns` defined?

Maybe elaborate that the `owns` config behaves in similar ways to the `owns` config in a versioned object?